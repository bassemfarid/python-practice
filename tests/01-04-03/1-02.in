2145.0
8829.0