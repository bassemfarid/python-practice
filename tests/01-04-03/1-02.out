9469102.5